bin/reth/src/lib.rs

```diff
@@ -15,7 +15,6 @@ pub mod p2p;
 pub mod prometheus_exporter;
 pub mod stage;
 pub mod test_eth_chain;
-pub mod util;
```

Comment:
I would do `pub use reth_cli_utils as utils` here so we can still access the utilities through `utils::`